cryptid_anchor/Anchor.toml

```diff
@@ -37,4 +38,4 @@ filename = "../fixtures/did-sol-idl-account.json"
 
 
 [scripts]
-test = "yarn run ts-mocha -p ./tsconfig.json -t 1000000 tests/**/*.ts"
+test = "yarn run ts-mocha --exit -p ./tsconfig.json -t 1000000 tests/**/*.ts"
```

Comment:
```      --exit                 Force Mocha to quit after tests complete  [boolean]
```
Why is this needed?

Reply:
without it, `anchor test` hangs for me after running tests. Not for you?